.github/workflows/e2e-tests.yml

```
@@ -14,7 +14,7 @@ jobs:
       fail-fast: false
       max-parallel: 10
       matrix:
-        project: [ 'default', 'acss', 'optimized-checkout', 'blik', 'becs' ]
+        project: [ 'default', 'isk', 'acss', 'optimized-checkout', 'blik', 'becs' ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)